AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -142,12 +142,17 @@ class MeetingModel: NSObject {
         self.selfName = selfName
         self.callKitOption = callKitOption
         self.meetingSessionConfig = meetingSessionConfig
-        if (meetingSessionConfig.meetingFeatures.videoMaxResolution == VideoResolution.videoDisabled) {
-            self.audioVideoConfig = AudioVideoConfiguration(videoMaxResolution: VideoResolution.videoDisabled)
-        } else if (meetingSessionConfig.meetingFeatures.videoMaxResolution == VideoResolution.videoResolutionHD) {
-            self.audioVideoConfig = AudioVideoConfiguration(videoMaxResolution: VideoResolution.videoResolutionHD)
+        if (meetingSessionConfig.meetingFeatures.videoMaxResolution == VideoResolution.videoDisabled
```

Comment:
Since we are passing `audioVideoConfig` as `parameter` in `init` already, I prefer to have the whole `if...else...` logic outside/before the `init`; or remove it in `init`.

Reply:
So I think the reason calling init for a second time is `videoMaxResolution` can only be determined after initialization of `audioVideoConfig`. I updated `videoMaxResolution` to a variable in the new revision so that `audioVideoConfig` does not have to be re-initialized but just updating video max resolution.

Reply:
Made another revision to refactor audio video config parameter passing and initialization.